src/test/scala/cromwell/CromwellSpec.scala

```diff
@@ -2,8 +2,40 @@ package cromwell
 
 import akka.actor.ActorSystem
 import akka.testkit.{TestKit, ImplicitSender, DefaultTimeout}
+import cromwell.binding._
+import cromwell.binding.values.{WdlString, WdlValue}
 import org.scalatest.{BeforeAndAfterAll, Matchers, WordSpecLike}
 
+object CromwellSpec {
+  val Config =
+    """
+      |akka {
+      |  loggers = ["akka.testkit.TestEventListener"]
+      |  loglevel = "DEBUG"
+      |  actor.debug.receive = on
+      |}
+    """.stripMargin
+
+  val HelloWdl =
```

Comment:
Nothing particular to the Hello testcase belongs here, this was intended to be a generic Spec class.